### ISSUE TITLE
Connect user class to user account data

### ISSUE THREAD

**Issue by JeremySanchez1** [uprm-inso4117-2023-2024-s2/semester-project-study-pet#92]
To securely connect a User class https://github.com/uprm-inso4117-2023-2024-s2/semester-project-study-pet/issues/56 with a User Account system, ensuring that users can register, log in, and access their accounts through proper authentication mechanisms. This integration should protect user data and ensure that only authenticated users can access their accounts.

Urgency: 3/5

Difficulty: 3/5

**Comment by irsaris** [comment]
Hello, are you still interested in doing this task?

**Comment by JeremySanchez1** [comment]
No, I wish to work on other areas, I'll be adding tasks shortly.

**Comment by irsaris** [comment]
@Adhel-J suggest scoring

**Comment by Adhel-J** [comment]
same as the comment above 
3/5 urgency
3/5 difficulty
